### DISCUSSION TITLE
Icons from this decade

### DISCUSSION THREAD

**Discussion by devcomputer** [JulianPrieber/littlelink-custom#789] [Feature requests]
To my suprise the phone icon is represented by a rotary device which is also said to be an ancient communication device. replacing those icons to something from the last decades is kinda hard.
Can we can an option to load icon sets or atleast get icons that a majority of users can understand?

**Comment by JulianPrieber** [comment]
![e](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExM3J5b2poMzNrMWgzb2p1c251c2EzM3FrZGI3NjdhdnI2cHNrMGdnMiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/GrUhLU9q3nyRG/giphy.webp)

**Comment by JulianPrieber** [comment]
There already is an option to load custom icon sets for themes.

**Reply by nightstryke** [reply]
Well that's great for themes, but what about custom icons for for custom links? That would be much more useful.
